x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -255,13 +255,13 @@ const getColumns = ({
     name: i18n.translate('xpack.csp.rules.rulesTable.mutedColumnLabel', {
       defaultMessage: 'Enabled',
     }),
-    width: '10%',
+    width: '5%',
```

Comment:
same here, both of there columns present a UI component with a set width. a switch and a checkbox, they can both use pixels